### ISSUE TITLE
Possible bug and/or feature request

### ISSUE THREAD

**Issue by Luminlyre** [streetpea/chiaki4deck#270]
First thank you for all the hard work you all have done on this project, it truley is amazing. 

I have a lcd steam deck and have been trying to figure out this issue for at least a week now. While streaming my PS5 to my deck using chiaki4deck, I am getting an itermittent Wifi notification pop up in the top right corner of my screen at random times. It looks like a blue bubble with a Wifi logo and a slash through it. My issue is I cannot find a way to stop it from happening regardless of trying everything I have found online to make my data stream more stable as well as configuring my network for the fastest possible connection. I am guessing it has do with the Steam Deck signal instability issue with their hardware flagging/triggering an unstable warning, however my stream audio and video are perfect with bitrates 15000 up to 30000.

Is it possible to disable this icon or add an option to disable it as a feature? The latest chiaki flatpak from what I tested does not have this issue, but then I lose automation functionality. Maybe I am overlooking something and the notification has nothing to do with chiaki itself.

Any help or guidance is greatly appreciated and thank you again for all your hard work. You guys rock!

**Comment by Destreyf** [comment]
@Luminlyre Are you on a beta release of SteamOS?  I noticed a recent issue while trying to get Chiaki4Deck working remote where my actual wifi signal was dropping and reconnecting, rolling back to a stable release stopped the issue, I then went back to beta and haven't seen it drop but I haven't had much time to test.

The specific behavior I noticed was in desktop mode, I would get a popup complaining that it lost connection and then it would immediately dismiss that notification and say reconnected.

I am on an oled steamdeck though, so maybe it was something specific to my unit, or maybe just even some failed software/firmware update.

**Comment by Luminlyre** [comment]
Hi and thanks for the reply! 
I haven't tried the beta channel at all since I got my deck (about 2 months ago) but it will be worth me looking into to see if that may resolve the issue. I haven't tested to see if chiaki4deck causes this in desktop mode only in game mode. However, observing my wifi in desktop mode, the signal will go from a solid 100% to anywhere from 95% - as low as 60% every 3 seconds roughly. 

Sounds like the same behavior you are experiencing, but my stream is still top notch and I notice no lag ir pixelation when it happens. Basically trying to see if there is a way to turn off the notifications completely as it is a bit distracting while playing games to see it pop up and go away almost immediately. 

The last release of chiaki 2.2.0 (not 4 deck) does not have this issue. Currently trying to figure out how to auto launch my PS5 using that version until I can figure out a way around removing the notification. 

I will give the beta channel a try and see if it resolves, thanks again!

**Comment by deedeeped** [comment]
I’m in the same boat. I’ve gotten my streaming experience to be great now; however, even without noticeable dropped packets, that WiFi icon will intermittently pop up when streaming via chiaki4deck. It’s nothing major, by any stretch, but it might be helpful to have it as an optional indicator enabled via toggle. Again, this is nothing horrible, but simply would be a nice UI adaptation.

**Comment by streetpea** [comment]
This symbol pops up when averagePacketLoss > 0.03. This means packets are being lost. The stream is resistant to packet loss errors so if it is just slightly at this mark you may not notice any stream degradation. I can certainly make this adjustable or allow you to turn it off if it's resulting in being more annoying than anything.

**Comment by deedeeped** [comment]
> Good afternoon and thank you for replying!
> 
> If that is something you would be willing to do and it isn't too much trouble that would be amazing! I cannot stress enough how amazing this program is and has been, I use it daily. You have done a fantastic job so far and I know I don't just speak for myself but many others in how much we appreciate the time and work that goes into this.
> 
> I assumed it had something to do with the network and was flagging chiaki4deck to register the instability. From what I have been reading online, network functions on the deck itself have been its weakpoint so that would explain some packet loss.
> 
> That being said, when there are packets being lost, they recover so quickly that I don't ever really notice and seems to be just long enough to flag the notification. I assume this wasn't present in the base version of chiaki as I did notice issues with the stream from time to time but never got a pop-up. Having it there is great for troubleshooting but more annoying if the stream is smooth and it shows up randomly.
> 
> Thank you again for the reply and consideration, keep up the great work and I look forward to the continued growth of chiaki4deck. The improvements you all have made over the original chiaki are extremely welcome to have and are a boon to gamers everywhere. Better than ps remote play hands down. 😊

Ditto to everything this person said. Thank you for all of your hard work! It’s made my Steam Deck the ultimate device.

**Comment by wx40217** [comment]
Me too. I was surprised the first time it happened because I didn't think it should happen with WiFi6 performance anyway. I'm leaning towards it being a problem with some configuration or profiles, because when a lag occurs, rebooting once or a number of times effectively fixes the problem, at least on my machine ,which means that my network is capable of loading the streaming requirements properly.

**Comment by Luminlyre** [comment]
I wanted to stop in and say thank you @streetpea for adding the proposed option into the chiaki4deck program.

I tested it for a few hours last night and a little bit today after I got home from church and it is working flawlessly. It is so incredibly cool to be able to play my playstation on deck while others are using the television. You sir are a gentleman and a scholar!

Thank you again for all your hard work, as well as the others who have contributed to this projecf. You folks are amazing and I respect you for all you do. I will close out this issue as resolved. Be blessed everyone!
